.env.example

```diff
@@ -108,6 +108,11 @@ DROPBOX_OAUTH_SECRET=
 WUNDERLIST_OAUTH_KEY=
 WUNDERLIST_OAUTH_SECRET=
 
+EVERNOTE_OAUTH_KEY=
+EVERNOTE_OAUTH_SECRET=
+# Set to true in development, false in production
```

Comment:
They might want to use it, and possibly also use a regular account in development (to test web clippings, which aren't available in sandbox). So I've just removed the comment